app/src/main/java/com/github/alfabravo2013/readyforexams/domain/login/LoginRemoteDataSource.kt

```diff
@@ -0,0 +1,33 @@
+package com.github.alfabravo2013.readyforexams.domain.login
+
+import android.util.Log
+import com.github.alfabravo2013.readyforexams.util.Result
+import com.google.firebase.auth.FirebaseAuth
+import com.google.firebase.auth.FirebaseUser
+import com.google.firebase.auth.ktx.auth
+import com.google.firebase.ktx.Firebase
+
+class LoginRemoteDataSource {
+    private val auth: FirebaseAuth by lazy { Firebase.auth }
+
+    private var currentUser: FirebaseUser? = auth.currentUser
```

Comment:
Don't need to do something for now but remember I said that our 3rd party should be inside a networking module? Firebase and all our networking should be there.